### ISSUE TITLE
Make Remux feature's compatibility a default download feature?

### ISSUE THREAD

**Issue by Sirocco745** [imputnet/cobalt#724]
### describe the feature you'd like to see

The new Remux feature is meant to "often fix compatibility issues with old software", and from my experience, it definitely works. Would it be possible to add an option to do something like "Download Remux'd File" instead of downloading **then** Remuxing? If this isn't possible or easy to code, why so? I'm genuinely curious about how these kinds of things work, and I'd like to know more.

### additional context

I've recently been having an issue with my Windows 11 laptop where, after trying to play a 3+ hour video **one time,** my laptop no longer properly registers the track length of video files and as a result doesn't allow me to skip through videos. Included are screenshots of this on:

Media Player
![image](https://github.com/user-attachments/assets/928a85d6-1a68-4951-b61d-3939100bad21)
and
![image](https://github.com/user-attachments/assets/601cd755-3f22-4ce2-b8d9-b34f3ae9008b)
Photos

The issue was persistent on all media player programs I tried, including VLC Media Player. Remuxing video files restores the capability to skip through videos for me.

**Comment by synzr** [comment]
remux video/audio is not necessary for many supported sources

**Comment by Sirocco745** [comment]
> video/audio remux is not necessary for many supported sources

Agreed, video/audio compatibility is pretty reliable nowadays. I'm not asking for all downloaded videos to be Remuxed by default, some sort of toggle switch in the downloads setting tab to turn off and on this would be sufficient to address this request. Admittedly, this request is probably annoying to implement and would affect a minority of users, but I thought it was worth a shot asking. Taking that into account, can I get a simple yes/no answer on this request? It's not crucial, which is why I'd just like some quick conformation.

**Comment by Sirocco745** [comment]
Also I'd just like to say something real quick, I really appreciate the speed with which you guys noticed and responded to this! It's not uncommon for me to be waiting hours for someone to hop on and check GitHub issues, so a big thanks for creating and maintaining cobalt.tools and doing a great job of it!

**Comment by alectrocute** [comment]
I would like to make a PR for this one.

**Comment by coderooo** [comment]
Hope to update this feature as soon as possible

**Comment by incohearant** [comment]
This feature would save me a ton of time. I use Cobalt to download .wav audio from Youtube, but the files won't play in finder (Mac) unless I put them through the remux tool, or re-export them using Logic Pro.

I get a lot of use out of Cobalt already, but if there was a setting to auto-remux all downloads, it'd be perfect. Either way, thanks to everyone who's worked on the project, it's very reliable and smooth.

**Comment by cysjunk** [comment]
Most video editing programs including modern VEGAS and After Effects don't seem to like Cobalt files either. I've brought this up before but it still persists. The way remuxing is implemented now also requires you to manually download the file unless you want to click on the site every 5 seconds.

My idea:
- Store unremuxed video in a temporary cache, auto-remux, serve video to user and delete cached unremuxed video automatically.
